layouts/partials/head.html

```diff
@@ -27,18 +27,11 @@
 <meta property="og:image" content="{{ index . 0 | replaceRE ".*src=\"(.+?)\".*" "$1" }}" />
 {{ end }}
 {{ end }}
-<script
-src="https://code.jquery.com/jquery-3.2.1.min.js"
-integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="
-crossorigin="anonymous"></script>
-<script
-src="https://code.jquery.com/ui/1.12.1/jquery-ui.min.js"
-integrity="sha256-VazP97ZCwtekAsvgPBSUwPFKdrwD3unUfSGVYrahUqU="
-crossorigin="anonymous"></script>
-<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/sweetalert/1.1.3/sweetalert.min.js"></script>
+<script src="{{ "js/jquery-3.2.1.min.js" | relURL }}"></script>
+<script src="{{ "js/jquery-ui.min.js" | relURL }}"></script>
+<script src="{{ "js/bootstrap.min.js" | relURL }}"></script>
+<script src="{{ "js/sweetalert.min.js" | relURL }}"></script>
 <script src="{{ "js/script.js" | relURL }}"></script>
 <script src="{{ "js/custom-jekyll/tags.js" | relURL }}"></script>
 {{ with .Params.js }}{{ range (split . ",") }}<script src="{{ (trim . " ") | relURL }}"></script><!-- custom js added -->
 {{ end }}{{ else }}<!-- no custom js detected -->{{ end }}
-<script async="" defer="" src="//survey.g.doubleclick.net/async_survey?site=6ip6epxiqjqn4qgzemchgtfziy"></script>
```

Comment:
@chenopis Are you OK with removing this for now and reconsidering how we approach site surveys for non-US visitors?

Reply:
Can you rename the js files to include the version number? Specifically, follow the `js/jquery-3.2.1.min.js` convention. It would be helpful for us to know which version of each component we are using.

Reply:
Yes, we have decided not to use Google Survey because it does not associate the results w/ the page URL. Go ahead and remove this.